assets/css/pre_fare_v2.scss

```diff
@@ -43,6 +45,10 @@
 
 @import "v2/pre_fare/simulation";
 
+// Don't import from solari... put in common place!
```

Comment:
I think we all three ended up using the Solari arrow, so this comment will be super helpful.